src/Util/GeometryUtil/GeometryUtil.js

```diff
@@ -0,0 +1,292 @@
+   * @param {String} projection A projection as EPSG code.
+   * @returns {ol.geom.Geometry | ol.Feature} A Feature or Geometry with the
+   * combined area of the polygons.
+   */
+  static union(polygons, projection = 'EPSG:3857') {
+    const geoJsonFormat = new OlFormatGeoJSON({
+      dataProjection: 'EPSG:4326',
+      featureProjection: projection
+    });
+    let invalid = false;
+    const geoJsonsFeatures = polygons.map((geometry) => {
+      const feature = geometry instanceof OlFeature
+        ? geometry
+        : new OlFeature({geometry});
+      if (feature.getGeometry().getType() !== 'Polygon') {
+        invalid = true;
+      }
+      return geoJsonFormat.writeFeatureObject(feature);
+    });
+    if (invalid) {
+      // Logger.warn('Can only create union of polygons.');
+      return undefined;
+    }
+    const unioned = union(...geoJsonsFeatures);
+    const feature = geoJsonFormat.readFeature(unioned);
+    if (polygons[0] instanceof OlFeature) {
+      return feature;
+    } else {
+      return feature.getGeometry();
+    }
+  }
+
+  /**
+   * Finds the difference between two polygons by clipping the second polygon from the first.
+   *
+   * @param {ol.geom.Geometry | ol.Feature} feature An ol.geom.Geoemtry or ol.Feature
+   * @param {ol.geom.Geometry | ol.Feature} feature An ol.geom.Geoemtry or ol.Feature
+   * @param {String} projection A projection as EPSG code.
+   *
+   * @returns {ol.geom.Geometry | ol.Feature} A Feature or Geometry with the area
+   *  of polygon1 excluding the area of polygon2.
+   */
+  static difference(polygon1, polygon2, projection = 'EPSG:3857') {
+    const geoJsonFormat = new OlFormatGeoJSON({
+      dataProjection: 'EPSG:4326',
+      featureProjection: projection
+    });
+    const feat1 = polygon1 instanceof OlFeature ? polygon1
+      : new OlFeature({
+        geometry: polygon1
+      });
+    const feat2 = polygon2 instanceof OlFeature ? polygon2
+      : new OlFeature({
+        geometry: polygon2
+      });
+    const geojson1 = geoJsonFormat.writeFeatureObject(feat1);
+    const geojson2 = geoJsonFormat.writeFeatureObject(feat2);
+    const intersection = difference(geojson1, geojson2);
+    const feature = geoJsonFormat.readFeature(intersection);
+    if (polygon1 instanceof OlFeature && polygon2 instanceof OlFeature) {
+      return feature;
+    } else {
+      return feature.getGeometry();
+    }
+  }
+
+  /**
+   * Takes two polygons and finds their intersection.
+   *
+   * @param {ol.geom.Geometry | ol.Feature} feature An ol.geom.Geoemtry or ol.Feature
+   * @param {ol.geom.Geometry | ol.Feature} feature An ol.geom.Geoemtry or ol.Feature
+   * @param {String} projection A projection as EPSG code.
+   *
+   * @returns {ol.geom.Geometry | ol.Feature} A Feature or Geometry with the
+   * shared area of the two polygons or null if the polygons don't intersect.
+   */
+  static intersection(polygon1, polygon2, projection = 'EPSG:3857') {
```

Comment:
See above comment regarding generality.

Reply:
`turf.js` only supports difference for intersect.